Drop-In/src/main/java/com/braintreepayments/api/dropin/adapters/AvailablePaymentMethodNonceList.java

```diff
@@ -0,0 +1,59 @@
+package com.braintreepayments.api.dropin.adapters;
+
+import android.content.Context;
+
+import com.braintreepayments.api.dropin.DropInRequest;
+import com.braintreepayments.api.dropin.utils.PaymentMethodType;
+import com.braintreepayments.api.models.CardNonce;
+import com.braintreepayments.api.models.Configuration;
+import com.braintreepayments.api.models.GooglePaymentCardNonce;
+import com.braintreepayments.api.models.PayPalAccountNonce;
+import com.braintreepayments.api.models.PaymentMethodNonce;
+import com.braintreepayments.api.models.VenmoAccountNonce;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+class AvailablePaymentMethodNonceList {
+
+    final private List<PaymentMethodNonce> items;
+
+    AvailablePaymentMethodNonceList(Context context, Configuration configuration, List<PaymentMethodNonce> paymentMethodNonces, DropInRequest dropInRequest, boolean googlePayEnabled, boolean unionpaySupported) {
+        items = new ArrayList<>();
+
+        for (PaymentMethodNonce paymentMethodNonce: paymentMethodNonces) {
+            boolean shouldAddPaymentMethod = false;
+
+            if (paymentMethodNonce instanceof PayPalAccountNonce && dropInRequest.isPayPalEnabled() && configuration.isPayPalEnabled()) {
```

Comment:
These long if statements are a little hard for me to read. Would it make sense to do something like:
```
if (paymentMethodNonce instanceof PayPalAccountNonce) {
    shouldAddPaymentMethod =  dropInRequest.isPayPalEnabled() && configuration.isPayPalEnabled();
} else if (paymentMethodNonce instanceof VenmoAccountNonce) {
    shouldAddPaymentMethod = dropInRequest.isVenmoEnabled() && configuration.getPayWithVenmo().isEnabled(context)
}
```

Reply:
Def yeah this works too. It should for sure cut down on the column count and make more readable.

Reply:
Yeah the shorter If statements are more readable, we've made this change.